src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js

```diff
@@ -165,4 +178,33 @@ export default class AircraftTypeDefinitionModel extends BaseModel {
         this.speed = null;
         this.capability = null;
     }
+
+    /**
+     * Build the string used for `#icaoWithWeightClass`
+     *
+     * @for AircraftTypeDefinitionModel
+     * @method _buildIcaoWithWeightClass
+     * @return {string}
+     * @private
+     */
+    _buildIcaoWithWeightClass() {
```

Comment:
instead of `IcaoWithWeightClass`, let's do something along the lines of `aircraftTypeString`. I know the use of `String` within the name itself is a no-no, and could be avoided if you prefer. But basically it comes down to `aircraftTypeString` or `aircraftTypeWithWeightClassPrefixAndEquipmentSuffix()` which would just be madness...